README.md

```diff
@@ -115,10 +115,12 @@ To run Documenso locally, you will need
 
 Want to get up and running quickly? Follow these steps:
 
-1. [Clone the repository](https://help.github.com/articles/cloning-a-repository/) it to your local device.
+1. [Fork this repository ](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) to your github account.
+
+After forking clone it using the below command:
```

Comment:
Could you please change this (and the one on line 159) to:

> After forking the repository, clone it to your local device by using the following command:

Reply:
Sure,I'll remove it.

Reply:
It doesn't appear that this line has been updated yet?

> After forking clone it using the below command:

Reply:
I had updated the changes.Can you tell me is it fine?.